client/src/components/classes/CampaignForm.js

```diff
@@ -1,215 +1,108 @@
-import React from "react";
-import {Grid, TextField} from "@material-ui/core/";
-import { withStyles } from "@material-ui/core/styles";
+import React, { useState, useEffect } from 'react';
+import { Grid, TextField, Card, Typography, Icon } from '@material-ui/core/';
+import { withStyles } from '@material-ui/core/styles';
+import axios from 'axios';
 
 const styles = theme => ({
   wrapper: {
     display: 'flex',
     flexWrap: 'wrap',
     alignItems: 'center',
     justifyContent: 'center',
-    margin: '1rem',
+    margin: '1rem'
+  },
+  cardList: {
+    border: '1px solid blue'
   },
   card: {
-    alignItems: 'center',
+    margin: 20,
```

Comment:
Maybe add `px` on your sizes at a later date just for readability purposes.

Reply:
Yes, good call!